docs/preview/logdecorations.md

```diff
@@ -0,0 +1,250 @@
+
+# Log decorations
+
+
+Task authors should be able to control how the log output is displayed to the end user.
+This outlines different decoration options that are available.
+
+## Collapse
+
+>This is a draft spec and is subject to change
+
+>Note that that if you log an error using ```##vso[task.logissue]error/warning message``` command (see [logging commands](https://github.com/Microsoft/azure-pipelines-tasks/blob/master/docs/authoring/commands.md) here) we will surface those errors in build view and when clicked , we will automatically jump to that particular line. If it's already part of a group, we will auto-expand the group.
+
+Task authors can mark any part of the log as a collapsible region using these decorations:
+
+Starting the collapsible region - `##[group:${optionalGroupName}:start]`
```

Comment:
what about `##[begingroup<:name>]` and `##[endgroup<:name>]` ?

Reply:
I like that much better actually, pushed!

Reply:
or `startgroup` instead of `begingroup`?

Reply:
Personally I like `startgroup` but don't feel strongly